packages/playwright/src/reporter.ts

```diff
@@ -27,10 +27,18 @@ export function getMetadataFilePath(workerIndex = 0) {
 }
 
 function extractErrorMessage(error: TestError) {
-  const errorMessageLines = removeAnsiCodes(error.message)!.split("\n");
-  let stackStart = errorMessageLines.findIndex(l => l.startsWith("Call log:"));
-  stackStart = stackStart == null || stackStart === -1 ? 10 : Math.min(stackStart, 10);
-  return errorMessageLines.slice(0, stackStart).join("\n");
+  if (error.message) {
+    // Error message. Set when [Error] (or its subclass) has been thrown.
+    const errorMessageLines = removeAnsiCodes(error.message)!.split("\n");
```

Comment:
Is `removeAnsiCodes(...)!` safe? If that method says it returns null/undefined, seems like we should check for that case?

Reply:
It is. it only returns undefined if the input parameter is undefined which we guarded for above.

Reply:
I reworked this to remove the assert to be safer though. Thanks for the nudge!

Reply:
> It is. it only returns undefined if the input parameter is undefined which we guarded for above.

Oooh! Makes sense. Seems like the ideal type for that would be `(value: string) => string)`

> I reworked this to remove the assert to be safer though

Thanks!